### PR TITLE
Added Agora Recording API Reference for CPP

### PR DESCRIPTION
**Pull request by lynnfirst**

### CODE REVIEW

File: docsets/Agora_Recording_CPP/docset.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+{
+    "name": "Agora C++",
```
Comment (Kapeli) [inline docsets/Agora_Recording_CPP/docset.json]: A docset named "Agora C++" already exists.

File: docsets/Agora_Recording_CPP/docset.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "version": "2.3.3",
+    "archive": "Agora_CPP.tgz",
```
Comment (Kapeli) [inline docsets/Agora_Recording_CPP/docset.json]: The archive you added is called "Agora_Recording_CPP.tgz".

Comment (lynnfirst) [reply docsets/Agora_Recording_CPP/docset.json]: Yes. The newly-added API reference is for a different product.

**Comment by Kapeli** [comment]
There's an "Agora Recording Java" docset. Why not name this one "Agora Recording C++"?

File: docsets/Agora_Recording_CPP/docset.json (hunk 2 above)
Comment (lynnfirst) [reply docsets/Agora_Recording_CPP/docset.json]: Sorry, I've already updated the information.

**Comment by Kapeli** [comment]
Please change the name to "Agora Recording C++"

**Comment by lynnfirst** [comment]
Done.

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**